helm_deploy/manage-my-prison/values.yaml

```diff
@@ -64,9 +64,25 @@ generic-service:
     102pf: "213.121.161.112/28"
     health-kick: "35.177.252.195/32"
     mojvpn: "81.134.202.29/32"
-    # cloudplatform-live1-1: "35.178.209.113/32"
-    # cloudplatform-live1-2: "3.8.51.207/32"
-    # cloudplatform-live1-3: "35.177.252.54/32"
+    quantum1: "62.25.109.197/32"
+    quantum2: "212.137.36.230/32"
+    quantum3: "195.92.38.16/28"
+    dxc_webproxy1: "195.92.38.20/32"
+    dxc_webproxy2: "195.92.38.21/32"
+    dxc_webproxy3: "195.92.38.22/32"
+    dxc_webproxy4: "195.92.38.23/32"
+    ark-nps-hmcts-ttp1: "195.59.75.0/24"
+    ark-nps-hmcts-ttp2: "194.33.192.0/25"
+    ark-nps-hmcts-ttp3: "194.33.193.0/25"
+    ark-nps-hmcts-ttp4: "194.33.196.0/25"
+    ark-nps-hmcts-ttp5: "194.33.197.0/25"
+    digitalprisons1: "52.56.112.98/32"
+    digitalprisons2: "52.56.118.154/32"
+    cloudplatform-live1-1: "35.178.209.113/32"
```

Comment:
Why would live-1 need access to this app?

Reply:
Good point. I'll remove them